docs/install-1.7.md

```diff
@@ -207,9 +226,9 @@ make test-e2e
 
 # Summary
 
-Before installing the helm chart, run the script in contrib by
-`source`ing it, to define all of the necessary variables.
+Before installing the helm chart, run the script in the `contrib` directory
+by `source`ing it. From the root of this repository, run this command:
 
 ```shell
-source /contrib/svc-cat-apiserver-aggregation-tls-setup.sh
+source ./contrib/svc-cat-apiserver-aggregation-tls-setup.sh
 ```
```

Comment:
We don't really want to run this script as it would overwrite any custom certs the user copied from their cloud environment as detailed in the steps above.  I believe all the steps above around the certs and TLS are executed with defaults by the svc-cat-apiserver-aggregation-tls-setup.sh script.   I realize this content was not added by his PR, but at some point this summary should be removed or replaced with an accurate summary.

Reply:
I agree. I'd just like to get this content into the right place for now and then improve in follow-ups. Cool with you?

Reply:
You're in the docs, why not fix them now?

Reply:
I will fix them, I just want to keep this PR's scope down as much as possible (it's already big).